### PR TITLE
feat: add MA sign in styling

### PR DESCRIPTION
**Pull request by cliu02**
# Pull Request Template

## Issue Overview

This PR addresses #3758. 

- [X] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

As part of the Mandated Accounts MVP, this PR applies the updated Figma form styles to the sign in page (https://github.com/bloom-housing/bloom/issues/3758). 

## How Can This Be Tested/Reviewed?

To review this PR:

1. Go to the `/sign-in` page.
2. Compare the form styling for this page with the [Figma](https://www.figma.com/file/ihaZA5QibRRJFyBdARKn55/User-Accounts?node-id=2591%3A4012&mode=dev) and ensure consistency.

Note: This ticket does not include changing the button styling for "Sign in" and "Create an account" or the show/hide password checkbox. Please let me know if you notice any other styling differences that should be accounted for in this ticket.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/public/styles/sign-in.module.scss
Comment (emilyjablonski) [inline sites/public/styles/sign-in.module.scss:2]: We're moving away from using tailwind, so these should be added as vanilla CSS or using seeds variables.

**Comment by emilyjablonski** [comment]
Hey @cliu02! In order to be able to have a core set of styles here and a different set of styles in Doorway, we need to be pulling in Seeds components where possible as part of this work. If you check out the ticket, there's a link to a PR in the description you can use for reference that has some examples of shifting from the UIC FormCard to the Seeds Card. 

Outside of shifting to Card, I do think the main headings here are looking larger and bolder than they should be in core.

File: shared-helpers/src/views/sign-in/FormSignIn.tsx
Comment (emilyjablonski) [inline shared-helpers/src/views/sign-in/FormSignIn.tsx:54]: Same comment from the other PR that uptaking the seeds Card will require removing these and using sections, recommend checking out some of the examples :)

File: shared-helpers/src/views/sign-in/FormSignIn.tsx
Comment (ColinBuyck) [inline shared-helpers/src/views/sign-in/FormSignIn.tsx:46]: @emilyjablonski Correct me if I'm wrong, but I was interpreting your comment about translations to mean that this should stay as "Sign In" until a full set of translations are ready for the switch to lowercase

**Comment by ColinBuyck** [comment]
Still reviewing but curious to get your thoughts as well as (@emilyjablonski @cade-exygy) on this: In my account styling PR, I establish patterns for AccountCard which would cover the create account container and header, and a seeds custom icon map which would allow the profile icon to be consistently seeds. Could we put this PR on hold until my [PR](https://github.com/bloom-housing/bloom/pull/3791) gets merged in and then incorporate those patterns? It needs some minor updates and 1 more review but shouldn't be long

File: shared-helpers/src/views/sign-in/FormSignIn.tsx
Comment (emilyjablonski) [reply shared-helpers/src/views/sign-in/FormSignIn.tsx:46]: We can't release this feature without translations, so if we know we are getting translations before release we are good to add new strings - otherwise we shouldn't change any strings or add any.

File: shared-helpers/src/views/accounts/AccountCard.tsx
Comment (ColinBuyck) [inline shared-helpers/src/views/accounts/AccountCard.tsx:7]: Think this can be changed back since we identified the source of the error was file location

File: shared-helpers/src/views/accounts/CustomIcons.tsx
Comment (ColinBuyck) [inline shared-helpers/src/views/accounts/CustomIcons.tsx:2]: Same with this?

**Comment by ColinBuyck** [comment]
Couple other notes:
1) Seems like the horizontal spacing is a little off since the card section has a total of 56 px between padding and margin but the footer has 48px (which matches the Figma)
2) Do you mind rebasing with main so that this can be reviewed with Cade's recent changes on the sign-in copy?

File: shared-helpers/src/views/accounts/CustomIcons.tsx
Comment (ColinBuyck) [reply shared-helpers/src/views/accounts/CustomIcons.tsx]: Ah yeah you're right, it is still needed! My bad!

**Review by ColinBuyck — APPROVED** [review]
LGTM 🥥 once the rebase issues are resolved! Thanks for all of the quick responses and updates!

**Review by cade-exygy — APPROVED** [review]
AC passes

File: shared-helpers/src/views/multiselectQuestions.tsx
Comment (emilyjablonski) [inline shared-helpers/src/views/multiselectQuestions.tsx:224]: I wouldn't have expected to see this change here, was it made explicitly?

File: sites/partners/package.json
Comment (emilyjablonski) [inline sites/partners/package.json:60]: Curious about this change

**Comment by emilyjablonski** [comment]
Let me know if you need assistance w the build issues!

**Pull request merged**
